<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:pressed {
    background-color: rgb(216, 216, 216);
}</string>
  </property>
  <widget class="QLabel" name="listlogo">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel#listlogo{
	
	background-image: url(:/img/listlogo.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#logo{
	color: rgb(255, 255, 255);
	font: 15pt &quot;Forte&quot;;
	background-color:rgb(234, 46, 73)
}</string>
   </property>
   <property name="text">
    <string>SimpleMusic</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="bottomWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>509</y>
     <width>991</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bottomWidget
{
background-color: rgb(31, 31, 31);
}
</string>
   </property>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#photo{
	
	background-color: rgb(176, 200, 178);
	/*border-radius:20px;*/
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#name{
 	
	color: rgb(255, 255, 255);
	font: 7pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="singer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#singer{
 	
	color: rgb(255, 255, 255);
	font: 6pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>Singer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="process">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>467</width>
      <height>41</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::groove:horizontal {                                
     border: 2px solid #1f1f1f;    
	 border-radius:10px;                     
     height: 2px;                                           
     margin: 0px 0;                                         
     left: 12px; right: 12px; 
	                           
}                                                          

QSlider::add-page:horizontal
{  
	border-radius: 3px;                             
   background: rgb(51, 55, 69);
}

QSlider::sub-page:horizontal {
	border-radius: 3px;
    background: rgb(234, 46, 73);
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pre">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>13</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pre
{
		background-image: url(:/img/pre.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pause
{
	background-color: rgb(31, 31, 31);
	background-image: url(:/img/play.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>13</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#next
{
		background-image: url(:/img/next.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#time
{
	
	color: rgb(255, 255, 255);
	font: 6pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>00：00：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="length">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#length
{
	
	color: rgb(255, 255, 255);
	font: 6pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>00：00：00</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::groove:horizontal {                                
     border: 2px solid #1f1f1f;    
	 border-radius:10px;                     
     height: 2px;                                           
     margin: 0px 0;                                         
     left: 12px; right: 12px; 
	                           
}                                                          

QSlider::add-page:horizontal
{  
	border-radius: 3px;                             
   background: rgb(51, 55, 69);
}

QSlider::sub-page:horizontal {
	border-radius: 3px;
    background: rgb(119, 196, 211);
}</string>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumelogo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#volumelogo
{
	background-image: url(:/img/volume.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playstyle">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>24</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#playstyle
{
	background-image: url(:/img/single.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fullScreen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#fullScreen{
	background-color: rgb(31, 31, 31);
 	color: rgb(255, 255, 255);
	font: 7pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget
{

	
	font: 8pt &quot;微软雅黑&quot;;
}
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="homepage">
    <widget class="QScrollArea" name="homeScrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>671</width>
       <height>441</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea#homeScrollArea
{
	
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="homeScrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#homeScrollAreaWidgetContents
{
	
	background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="searchpage">
    <widget class="QRadioButton" name="nameRadioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
 	
	font: 8pt &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="singerRadioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
 	
	font: 8pt &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>Singer</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>671</width>
       <height>411</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="searchContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>411</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="managepage">
    <widget class="QLabel" name="musicListLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#musicListLabel
{
	
	font: 8pt &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>歌曲列表：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addMusic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#addMusic
{
	
	font: 8pt &quot;微软雅黑&quot;;
	border-radius:3px;
	background-color: rgb(132, 132, 132);
}</string>
     </property>
     <property name="text">
      <string>添加歌曲</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="videolistwidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>39</y>
       <width>671</width>
       <height>350</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65535</width>
       <height>65535</height>
      </size>
     </property>
     <property name="locale">
      <locale language="Corsican" country="France"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>350</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65535</width>
        <height>65535</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="settingpage">
    <widget class="QLabel" name="dirlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#dirlabel
{
	
	font: 8pt &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>歌曲目录:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#pathlineEdit
{
	
	font: 8pt &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>D:/video</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="videopage">
    <widget class="QWidget" name="videowidgepage" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>692</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="searchInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#searchInput
{
	
	color: rgb(150, 150, 150);
	font: 8pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>Search By Music Name Or Singer</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="musicsearch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#musicsearch
{
	background-image: url(:/img/search.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>150</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#leftWidget{

	background-color:rgb(234, 234, 234);

}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="home">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#home{
	
	font: 75 8pt &quot;微软雅黑&quot;;
	
}

QPushButton#home:hover{
	background-color: rgb(234, 215, 218);
	flate
}</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="search">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#search{
	
	font: 75 8pt &quot;微软雅黑&quot;;
}

QPushButton#search:hover{
	background-color: rgb(234, 215, 218);
}</string>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="manage">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#manage{
	
	font: 75 8pt &quot;微软雅黑&quot;;
}

QPushButton#manage:hover{
	background-color: rgb(234, 215, 218);
}</string>
      </property>
      <property name="text">
       <string>MANAGE</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setting">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#setting{
	
	font: 75 8pt &quot;微软雅黑&quot;;
}

QPushButton#setting:hover{
	background-color: rgb(234, 215, 218);
}</string>
      </property>
      <property name="text">
       <string>SETTING</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="video">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#video{
	
	font: 75 8pt &quot;微软雅黑&quot;;
}

QPushButton#video:hover{
	background-color: rgb(234, 215, 218);
}</string>
      </property>
      <property name="text">
       <string>VIDEO</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitbutton">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#exitbutton{
	
	font: 75 8pt &quot;微软雅黑&quot;;
}

QPushButton#exitbutton:hover{
	background-color: rgb(234, 215, 218);
}</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>150</width>
     <height>449</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="rightWidght">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>449</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#rightWidght
{
	background-color:rgb(234, 234, 234);
}</string>
    </property>
   </widget>
  </widget>
  <zorder>leftWidget</zorder>
  <zorder>listlogo</zorder>
  <zorder>logo</zorder>
  <zorder>bottomWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>searchInput</zorder>
  <zorder>musicsearch</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitbutton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
